### DISCUSSION TITLE
Translations through Weblate

### DISCUSSION THREAD

**Discussion by nichwall** [advplyr/audiobookshelf#3076] [General]
Hey there!

Thank you to all who have helped to translate the web client and mobile apps in the past. We wanted to let all of you know we have switched to using [Weblate](https://hosted.weblate.org/engage/audiobookshelf/) for localizating the ABS web client and mobile apps. We are still waiting for approval for their libre hosting tier (can take up to 2 months), but everything is set up and PRs from Weblate are working. We will make another post on this discussion if anything changes.

If you would like to help out with the localization efforts, all you need to do is create an account on Weblate (can sign in using your GitHub account) and then you can start translating. All strings are currently marked as needing review after the initial import, but once this initial review process is completed managing updates will be much easier.

If you are on the ABS Discord, we have also created a new `translator` role so you can be pinged if there are a lot of new strings to translate. Please reach out on Discord if you would like to be added to the role.

<a href="https://hosted.weblate.org/engage/audiobookshelf/">
<img src="https://hosted.weblate.org/widget/audiobookshelf/multi-auto.svg" alt="Translation status" />
</a>

Special thanks to (hopefully I got everyone):
- Bengali: @soaibsafi
- Bulgarian: @lembata
- Simplified Chinese: @springsunx @Jafeyyu
- Traditional Chinese: @den13501 
- Croatian: @Smoukus
- Czech: @radekmuhlfeit2 @Plazec @arcao
- Dutch: @92Kev 
- Estonian: @RasmusKoit 
- French: @Machou @tomazed @glacasa @Fymyte
- German: @JBlond @Dalabad @Myticktack @lonezel @Teekeks @bloodscript @Hallo951 @Lionfox2 @jkuehnemundt @Demian98 @Smoukus @jannikgohr
- Hindi: @Dr-Blank 
- Gujarati: @Dr-Blank 
- Hebrew: @arcmagedr 
- Hungarian: @megamegax 
- Italian: @burghy86
- Lithuanian: @petras-sukys 
- Norwegian: @husjon 
- Polish: @konradorlinski @Bogumio
- Portuguese (Brazil): @pmangro 
- Russian: @Nab0y 
- Spanish: @diamondtipdr @DownloadableFox @MarshDeer @apineiro97
- Swedish: @ScuttleSE 
- Ukrainian: @cor-bee 
- Vietnamese: @saigonviral

**Comment by Machou** [comment]
If only there was more choice like this in the world of translation :) top!

**Comment by JBlond** [comment]
The project URL: https://hosted.weblate.org/projects/audiobookshelf/
